smaug/core/network_builder.cpp

```diff
@@ -55,6 +55,8 @@
 using namespace smaug;
 using namespace std;
 
+const std::string kFakeParentName = "N/A";
```

Comment:
If we're going to do this, we need a constant for -1 as well, but as I mentioned in the overall comment, I don't like the direction this is going.